pkg/helm/handlers/handlers.go

```diff
@@ -20,6 +21,14 @@ var (
 type HelmHandlers struct {
 	ApiServerHost string
 	Transport     http.RoundTripper
+
+	// helm action configurator
+	GetActionConfigurations func(string, string, string, *http.RoundTripper) *action.Configuration
```

Comment:
there is no need to make these functions public, see https://github.com/akashshinde/console/pull/3